src/Cli/Utils.cs

```diff
@@ -186,26 +186,26 @@ public static bool TryParseMappingDictionary(IEnumerable<string> mappingList, ou
         /// <summary>
         /// Returns true if the api path contains any reserved characters like "[\.:\?#/\[\]@!$&'()\*\+,;=]+"
         /// </summary>
-        /// <param name="apiPath">path prefix for rest/graphql apis</param>
+        /// <param name="uriComponent">Path prefix/base route for REST/GraphQL APIs.</param>
         /// <param name="apiType">Either REST or GraphQL</param>
-        public static bool IsApiPathValid(string? apiPath, ApiType apiType)
+        public static bool IsURIComponentValid(string? uriComponent, ApiType apiType, string apiProperty)
```

Comment:
```suggestion
        public static bool IsUriComponentValid(string? uriComponent, ApiType apiType, string apiProperty)
```
nit: not necessary to have uppercase URI.